bigtable/examples/bigtable_samples_instance_admin.cc

```diff
@@ -98,21 +141,25 @@ int main(int argc, char* argv[]) try {
   //! [connect instance admin]
 
   if (command == "create-instance") {
-    CreateInstance(instance_admin, project_id, instance_id);
+    CreateInstance(instance_admin, argc, argv);
   } else if (command == "list-instances") {
-    ListInstances(instance_admin);
+    ListInstances(instance_admin, argc, argv);
   } else if (command == "get-instance") {
-    GetInstance(instance_admin, project_id, instance_id);
+    GetInstance(instance_admin, argc, argv);
   } else if (command == "delete-instance") {
-    DeleteInstance(instance_admin, instance_id);
+    DeleteInstance(instance_admin, argc, argv);
   } else if (command == "list-clusters") {
-    ListClusters(instance_admin);
+    ListClusters(instance_admin, argc, argv);
   } else {
-    std::cerr << "Unknown command: " << command << std::endl;
-    print_usage();
+    std::string msg("Unknown_command ");
```

Comment:
Why not retain the original format of the error, i.e., `"Unknown command: " + command`?

Reply:
Fixed.